boot.php

```diff
@@ -15,7 +23,12 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+if(file_exists(getcwd().'/'.$argv[1])){
+    $filename = getcwd().'/'.$argv[1];
+} else{
+    $filename = $argv[1];
```

Comment:
When you pass `~/roms/drmario.gb` the Shell itself, not PHP, changes it to the absolute path `/home/username/roms/drmario.gb`. If it exists the script will execute the ROM.

If you're in `/home/username` and you pass `roms/drmario.gb` the PHP already known what is your current working directory, so if `/home/username/roms/drmario.gb` exists the script will execute the ROM.

But you don't need to trust me, just make a test.

```bash
[~] $ php-gameplay ~/roms/drmario.gb
[~] $ php-gameplay roms/drmario.gb
```

That's why since the beginning I said that  `getcwd().'/'.` is unnecessary.